JumpTheQueue/index.asciidoc

```diff
@@ -0,0 +1,41 @@
+= Tutorial for the Jump The Queue Application
+====
+Jump The Queue is a tutorial application
```

Comment:
I think we should write a little more here. For example, that the Jump The Queue application consists of a Java backend and an Angular frontend and that it is based on the devonfw architecture.
You should also keep the name conistent. Here you use 'Jump The Queue' and in the next line 'JumpTheQueue'.